unittests/rpc/handlers/AccountTxTests.cpp

```diff
@@ -1441,6 +1511,10 @@ TEST_F(RPCAccountTxHandlerTest, NFTTxs_API_v2)
     )
         .Times(1);
 
+    auto const ledgerInfo = CreateLedgerInfo(LEDGERHASH, 11);
+    ON_CALL(*rawBackendPtr, fetchLedgerBySequence).WillByDefault(Return(ledgerInfo));
+    EXPECT_CALL(*rawBackendPtr, fetchLedgerBySequence).Times(transactions.size());
```

Comment:
This could be done in one line
```cpp
EXPECT_CALL(*rawBackendPtr, fetchLedgerBySequence).Times(transactions.size()).WillRepeatedly(Return(ledgerInfo));
```